python/ray/tune/BUILD

```diff
@@ -197,7 +197,7 @@ py_test(
     size = "medium",
     srcs = ["tests/test_progress_reporter.py"],
     deps = [":tune_lib"],
-    tags = ["team:ml", "exclusive"],
+    tags = ["team:ml", "exclusive", "test_progress_reporter"],
```

Comment:
Instead of this, maybe just use `pytest.mark.skipif(os.environ.get("AIR_VERBOSITY", "1"))` to avoid cluttering too many bazel tags.